src/Humanizer/Localisation/Formatters/DefaultFormatter.cs

```diff
@@ -88,22 +88,34 @@ private string GetResourceForTimeSpan(TimeUnit unit, int count)
         /// <summary>
         /// 
         /// </summary>
-        /// <param name="resourceKey"></param>
+        /// <param name="resourceKey">The resource key.</param>
         /// <returns></returns>
+        /// <exception cref="ArgumentException">If the resource not exists on the specified culture.</exception>
         protected virtual string Format(string resourceKey)
         {
-            return Resources.GetResource(GetResourceKey(resourceKey), _culture);
+            var resourceString = Resources.GetResource(GetResourceKey(resourceKey), _culture);
+
+            if (string.IsNullOrEmpty(resourceString))
+                throw new ArgumentException(string.Format("The resource object with key '{0}' was not found", resourceKey), nameof(resourceKey));
```

Comment:
You can use string interpolation.

    string.Format("The resource object with key '{0}' was not found", resourceKey)

->

    $"The resource object with key '{resourceKey}' was not found"